themes-default/slim/static/js/lib/http-vue-loader.js

```diff
@@ -0,0 +1,470 @@
+(function umd(root,factory){
```

Comment:
I still prefer a minimised version of this file, unless you believe it would be easier to debug issues?

Reply:
Yeah, I think for now it'll be easier.